apps/condo/pages/ticket/index.tsx

```diff
@@ -95,7 +106,9 @@ const TicketsPage: IPageWithHeaderAction = () => {
     )
 
     const [filtersApplied, setFiltersApplied] = useState(false)
-    const tableColumns = useTableColumns(sortFromQuery, filtersFromQuery, setFiltersApplied)
+
+    const useTableColumnsHook = useTableColumnsFromProps ? useTableColumnsFromProps : useTableColumns
```

Comment:
Maybe it's better to pass the columns inside?

Reply:
In 65811ec I took out the `TicketsPageContent` component, into which I pass `searchTicketsQuery`, `tableColumns` and props for sorting and filtering.
Thus, it will be possible to reuse the logic and markup of `TicketsPage` with different `tableColumns` and `searchTicketsQuery`.